<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Максименко И-03</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>111</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Приветствие</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Введите имя:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton22">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ОК</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>122</x>
        <y>10</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Данил</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Сложение</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton222">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>+</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>-</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>*</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>/</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Время</string>
     </attribute>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>331</width>
        <height>71</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>8</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Таймер</string>
     </attribute>
     <widget class="QPushButton" name="Start">
      <property name="geometry">
       <rect>
        <x>94</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Старт</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Stop">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Стоп</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Reset">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Сброс</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Reset_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Установить время</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="Reset_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Установоить будильник</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>264</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Автор</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>311</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Максименко Д. В.
Гр. И-03
24.10.2023</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Майнкрафт</string>
     </attribute>
     <widget class="QPushButton" name="left">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="right">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="up">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>^</string>
      </property>
     </widget>
     <widget class="QPushButton" name="down">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>v</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="right_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>383</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
